src/test/java/domain/FixedNumbersGenerator.java

```diff
@@ -0,0 +1,17 @@
+package domain;
+
+import java.util.Set;
+
+public class FixedNumbersGenerator implements LottoGenerator {
```

Comment:
```suggestion
public class FixedLottoGenerator implements LottoGenerator {
```

ManualLottoGenerator도 테스트 해주세요!